data/common.yaml

```diff
@@ -0,0 +1,15 @@
+---
+svckill::ignore_defaults:
+  - auditd
+  - puppet
+  - puppetmaster
+  - crond
+  - sshd
+  - iptables
+  - ip6tables
+  - ebtables
+  - rc
+  - ^pe-.*
```

Comment:
the PE one I was explicitly told to use. The redhat-7 ones were to hit service template service names if they existed (which this type is not exactly handling all that well to start with, but then again neither does puppet... yay systemd!)